examples/kill_master_node.js

```diff
@@ -0,0 +1,35 @@
+// In this test we disable master node of replicaset
+// Master node is excepted to enter the inconsistent state
+
+import k6 from "k6";
+import khorne from "k6/x/khorne";
+
+export const options = Object(
+  {},
+  {},
+  {
+    vus: 1,
+    iterations: 1,
+  }
+);
```

Comment:
You can do it just like that:

```suggestion
export const options = {
  vus: 1,
  iterations: 1,
};
```